app/layout.tsx

```diff
@@ -45,6 +45,7 @@ export default function RootLayout({
             <Logo />
           </div>
           <div className="w-4/5 flex justify-between items-center p-3">
+            {/* <NavigationMenuDemo /> */}
```

Comment:
This was removed in the latest version you can just delete it.

Reply:
I have deleted it.